test/k8s-config/test-env.sh.yml

```diff
@@ -65,12 +65,10 @@ spec:
             value: '5'
 
           - name: CONJUR_APPLIANCE_URL
```

Comment:
Please rename to more generic name that is appropriate to OSS as well. Say CONJUR_URL

Reply:
https://app.zenhub.com/workspaces/moti-5d99d900491c060001c85cba/issues/cyberark/conjur-docs/591

Reply:
UPDATE: we will challenge this in a separate issue when we have more time as it's not severe. at this point i'm leaving it with `CONJUR_APPLIANCE_URL` as it might be required by the conjur-authn-client